### ISSUE TITLE
Notices are not full width for full-width sidebars.

### ISSUE THREAD

**Issue by Copons** [WordPress/gutenberg#14775]
**Describe the bug**
At widths where opening the sidebar would expand it full-width (such as a screen width of 850 pixels), admin notices will shrink to the size of a normal column. This makes the admin notice appear to be floating to the left side (it's full-width and fine, however, when the sidebar is closed).

**To Reproduce**
1. Resize the window enough so that the sidebar is full-width.
2. Close the sidebar and update the post.
3. Notice that the success notice is full-width.
4. Open the sidebar.
5. Check that the success notice occupies roughly half of the window width.

**Expected behavior**
That the notice occupied the entire width available.

**Screenshots**

<img width="863" alt="55373983-d030fe00-54bb-11e9-8c49-9a93c5da8583" src="https://user-images.githubusercontent.com/2070010/55413594-87338500-5561-11e9-8ba2-daafadf8cd73.png">
<img width="866" alt="55373989-d7580c00-54bb-11e9-9be4-9a737aee6268" src="https://user-images.githubusercontent.com/2070010/55413596-87338500-5561-11e9-9dd4-ec3d0757c425.png">

**Additional context**

This seems to happen because of this mixin:
https://github.com/WordPress/gutenberg/blob/495f54cadb6cf1aac4b531fa715e3afcf18729e7/assets/stylesheets/_mixins.scss#L294-L306
Which is compiled into:
```css
.edit-post-layout.is-sidebar-opened .components-notice-list {
  right: 280px;
}
```

Originally reported in https://github.com/Automattic/wp-calypso/issues/31945

**Comment by draganescu** [comment]
Hi @Copons I tried to reproduce in Chrome and wasn't able to, for me the sidebar covers the success notice.

**Comment by youknowriad** [comment]
Same, can't reproduce, did you try with the last Gutenberg release?

**Comment by Copons** [comment]
Hi all, thanks for testing!
This is a gif I've recorded a few minutes ago on a freshly pulled master (on fc20eeac75776fa2eb40497092551f5031dc9546):

![Apr-03-2019 11-09-05](https://user-images.githubusercontent.com/2070010/55471122-f44a2780-5600-11e9-86b2-d066814119e3.gif)

**Comment by youknowriad** [comment]
Mmm weird, still not able to reproduce personally. Which browser is this? Do you have some gutenberg plugins enabled?

**Comment by Copons** [comment]
Dammit, I feel really dumb.
My test site had the Guten plugin disabled, so even though I was building it on the latest commit, I was testing on the Core version. 🤦‍♂️ 

So yeah, you all are correct, I can't reproduce either.
I'll close this issue and report back that the issue will likely fix itself with 5.2.

Thanks for checking and sorry about this. 🙇
